css/properties.json

```diff
@@ -4766,7 +4766,7 @@
     "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/fill"
   },
   "fill-opacity": {
-    "syntax": "<alpha-value>",
+    "syntax": "<number> | <percentage>",
```

Comment:
maybe [<'opacity'>](https://developer.mozilla.org/en-US/docs/Web/CSS/opacity) or <opacity-value> is better?

see https://github.com/mdn/data/pull/784